Library/Formula/rubber.rb

```diff
@@ -11,18 +12,11 @@ class Rubber < Formula
     sha256 "e7480ca80262718ac62b26c3573c1b7e3091a2ed1766d7f33967f2620911fbc4" => :mountain_lion
   end
 
-  head "lp:rubber", :using => :bzr
+  depends_on "texinfo" => :build
+  depends_on :tex
```

Comment:
> I can submit an upstream issue. But what we want? Not to fail the build without tex installed?

I don't think there is a point to do that. `Rubber` is a LaTeX build system(similar to `latexmk` which is also available in `homebrew/tex`). I think this formula should be submit to `tex` tap.